Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,45 @@
+public class MockView: Viewable {
```

Comment:
Just because a method is private doesn't mean it should be tested, it just means that the private method's functionality should be tested through the public interface of the class.